.github/workflows/release.yml

```
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - master
+  schedule:
+    - cron: '0 * * * *'
```

Nitpicking here: I would suggest randomizing the starting minute
```suggestion
    - cron: '28 * * * *'
```
As recommended here https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule